frontend/src/metabase/search/components/InfoText/InfoText.tsx

```diff
@@ -63,33 +64,56 @@ export const InfoTextTableLink = ({ result }: InfoTextProps) => {
   }
 
   return (
-    <SearchResultLink key={label} href={link}>
+    <SearchResultLink isEnabled={isEnabled} key={label} href={link}>
       {label}
     </SearchResultLink>
   );
 };
 
-export const DatabaseLink = ({ database }: { database: Database }) => (
-  <SearchResultLink key={database.name} href={browseDatabase(database)}>
+export const DatabaseLink = ({
+  database,
+  isEnabled,
+}: {
+  database: Database;
+  isEnabled: boolean;
+}) => (
+  <SearchResultLink
+    key={database.name}
+    href={browseDatabase(database)}
+    isEnabled={isEnabled}
+  >
     {database.name}
   </SearchResultLink>
 );
-export const TableLink = ({ result }: { result: WrappedResult }) => {
+export const TableLink = ({
+  result,
+  isEnabled,
+}: {
+  result: WrappedResult;
+  isEnabled: boolean;
+}) => {
   const link = browseSchema({
     db: { id: result.database_id },
     schema_name: result.table_schema,
   });
 
   return (
     <>
-      <SearchResultLink key={result.table_schema} href={link}>
+      <SearchResultLink
+        isEnabled={isEnabled}
+        key={result.table_schema}
```

Comment:
1. Wrapping fragment is redundant
2. `key` is redundant (also in other components here)